s32/drivers/s32ze/BaseNXP/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 NXP
+# Copyright 2022,2024 NXP
```

Comment:
revert, no change in the file

Reply:
I updated the copyright

Reply:
copyright must be updated only if there was a (substantial) change in the file

Reply:
Exactly, I mean I reverted the copyright in the file that was not changed and updated the copyright as NXP copyright notice format in the changed files.